### PR TITLE
Api test for force payment + bugfix

### PR DESCRIPTION
**Pull request by Jakub89**
Add cluster integration tests for issue #146 , rest will be added in next pull request.

### CODE REVIEW

File: concent_api/core/payments/mock.py [hunk 1]
```diff
@@ -8,7 +9,7 @@ def get_client_eth_account():
     return True
 
 
-def get_number_of_eth_block(request = None):  # pylint: disable=inconsistent-return-statements
+def get_number_of_eth_block(request = None, _subtask_results_accepted_list = None, _list_of_transactions = None, _list_of_forced_payments = None):  # pylint: disable=inconsistent-return-statements
```
Comment (rwrzesien) [inline concent_api/core/payments/mock.py]: Please also change in `payments\base.py`. I see there are now two function calling `get_number_of_eth_block`. There should be only one, and second one should call `payment_summary()`. Please also make sure that both functions signature is correct in base and mock backends, and their usage in other code is correct.

Comment (Jakub89) [reply concent_api/core/payments/mock.py]: Okey, I see. Function `get_number_of_eth_block` is currently not used at all in our code, and `payment_summary` is used only once. Should I maybe remove `get_number_of_eth_block` completely for right now?

Comment (rwrzesien) [reply concent_api/core/payments/mock.py]: About removal, please discuss it with @kbeker .

This it not fixed yet:
1. `payment_summary` in base.py should call backend.payment_summary
2. `get_number_of_eth_block` in base.py should have only `backend, request` parameters.
3. `get_number_of_eth_block` in mock.py should have only `request` parameter.

Comment (Jakub89) [reply concent_api/core/payments/mock.py]: Done, and I'll talk to @kbeker when I'll got a chance :)

Comment (rwrzesien) [reply concent_api/core/payments/mock.py]: Ok, thanks.

**Pull request closed without merging**